imports/ui/components/ideas/BookmarkIdeaLink.jsx

```diff
@@ -41,19 +41,18 @@ export default class BookmarkIdeaLink extends Component {
   }
   linkClasses() {
     let classes = classNames(
-      'bookmark', 'right', {
-        'active': this.state.bookmarked
-      });
```

Comment:
'bookmark',  {
'active': this.state.bookmarked,
'edit': this.props.view,
'right': !this.props.view
 });
would be more readable